docs/build/dapp/10-faq.md

```diff
@@ -0,0 +1,20 @@
+---
+id: faq
+title: FAQ
+sidebar_label: FAQ
+---
+
+## How to test a WebSocket connection locally?
+
+Ensure that the Archethic wallet is running and listening on the correct port. You can use tools like wscat for testing.
+
+## Which browsers are supported?
+
+Currently, only Chrome is supported with the Archethic extension.
```

Comment:
perhaps "Currently, only browsers based on chromium (Google Chrome, Brave, ...) are fully supported with the Archethic extension."